docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
-* isolation (default|hyperv)   (Windows daemon only)
+* isolation (default|process|hyperv)   (Windows daemon only)
```

Comment:
@thaJeztah Well that was an interesting exercise over the past hour. It’s actually incredibly difficult and breaks layering in the daemon to not filter on “default”. The reason is that if no –exec-opt is specific on the daemon to say what the default is, and a container is created without a specific value, the value in the hostConfig structure on the daemon is deliberately blank. It’s only at execution time that the blank value is evaluated and calculated.

So to work out what blank means, I’d have to call into the exec driver to find out what the default is for that instance of the daemon which is horrible. IOW, I don’t think it actually makes sense to do this change.